microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -15,10 +15,10 @@ The web is constantly evolving to improve the user experience, security, and pri
 
 | Feature area | Status in Microsoft Edge | Additional information |  
 | --- | --- | ---:|  

```

Comment:
The table is right aligned. Can you left align it please?

Reply:
Global comment for the 2nd column. Can we change "starting in v82" to "(available in v82)."?

Reply:
>The table is right aligned. Can you left align it please?

I haven't changed the formatting of the table from what was originally there; I'm happy for it to be left aligned if that's better somehow, but it doesn't seem like a blocking issue.

>Global comment for the 2nd column. Can we change "starting in v82" to "(available in v82)."?

This seems less clear than the original (it implies that it's "available" not "required" and it suggests that it might only apply to 82, rather than applying to 83+.

Reply:
Thanks for asking. Left alignment for table data follows the MS Style Guidelines.

I suggest "(available in v82 or later)." to follow the MS Style Guide.

Reply:
The alignment was fixed in another PR.

Reply:
The suggestion was moved into a suggestion box.